docs/hooks.md

```diff
@@ -93,7 +93,7 @@ Available Hooks
 
     Alter the raw HTTP response before returning for parsing.
 
-    Parameters: `string &$headers`, `[array &$info]`
+    Parameters: `string &$headers`, `[array &$info]`, `[int|string $id]`
```

Comment:
With your changes in Curl.php $info and $id can also be NULL right?

Reply:
So, kinda. The `[]` indicates that they are optional parameters. **Technically** any hook could set any default value for those. The only thing that's different now compared to before for `$info` is that we now **have to** provide a default value on the call site.